tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -2192,6 +2192,18 @@ system_health/test_system_health.py::test_service_checker_with_process_exit:
       - "branch in ['internal-202012']"
       - "build_version.split('.')[1].isdigit() and int(build_version.split('.')[1]) <= 44"
 
+system_health/test_system_health_fan_direction.py:
+  skip:
+    reason: "There is no table SYSTEM_HEALTH_INFO in STATE_DB on kvm testbed, skip in PR testing"
+    conditions:
+      - "asic_type in ['vs']"
+
+system_health/test_system_health_led.py:
+  skip:
+    reason: "There is no table SYSTEM_HEALTH_INFO in STATE_DB on kvm testbed, skip in PR testing"
```

Comment:
Same as above

Reply:
> Same as above

Hi, @yutongzhang-microsoft ,

The skip reason is revised.